ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx

```diff
@@ -499,7 +498,8 @@ export const AttributesPage = withCurrentWorkspace() (
       const {node} = this.props;
       const {calculating, count, countError, form, loading, options} = this.state;
       const {formErrors, formValid} = this.validateForm();
```

Comment:
Why do we have formErrors and formValid? Seems like these could get out of sync in the validateForm function? Can you get away with 1 const like fromErrors? If formErrors is empty the the form is valid?

Reply:
We don't show an error for an empty input unless it's already been typed in, so it is possible for the form to be invalid and have an empty error array.